docs/commands/lint.md

```diff
@@ -111,9 +88,7 @@ If you try to execute the `lint` command without apis when your project doesn't
 
 By default, the CLI tool looks for the [Redocly configuration file](../configuration/index.md) in the current working directory. Use the optional `--config` argument to provide an alternative path to a configuration file.
 
-```bash
-redocly lint --config=./another/directory/config.yaml
-```
+`redocly lint --config=./another/directory/config.yaml`
```

Comment:
Isn't it better as before as a code block? (it is copy/paste-able as above, and people can also report problems with the code snippet as above)

Reply:
I think the general consensus was that really the whole page needs refactoring, but that this way is fine for now (and better than the wall of code samples). Happy to open a followup issue, but I'd like to focus on the example updates in this PR.

Reply:
I may look better (which is a matter of taste 💅) but I find it way harder to copy this way.
It's also inconsistent as some pages use code snippets and this one uses inline blocks.

If we decide to keep it as is can we open a ticket for refactoring this page?

Reply:
We definitely can! There's a lot more we could do if we wanted to invest some serious time in all the command pages IMO. Readability was my main aim on this pass, I think copying the config is more valuable than copying a short command that a user will need to edit anyway, so I picked this setup as the least worst option.

Reply:
I've added #1429 so we don't lose sight of this.